whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageParser.java

```diff
@@ -177,6 +177,7 @@ private void parseContents(@Nonnull final MailMessageBuilder messageBuilder, @No
     }
 
     private void parseContents(@Nonnull final MessageParts messageParts, @Nonnull final Part part, @Nullable final Part parentPart) throws MessagingException, IOException {
+        handleInvalidTypes(part);
         final ContentType contentType = new ContentType(part.getContentType());
```

Comment:
Could you use a `new ContentType("text/plain")` if it's text, rather than re-writing the message part header?

Reply:
If I don't fix this in the message then line 173 will throw an error
we are doing new ContentType(message.getContentType()) there as well
I don't want to do this check everywhere...if the message contentType is not parseable by Jakarta library we should fix the message directly. No if/else everywhere when we are doing getContentType()

Reply:
OK